The application crashes when accessing the Rate/Vote giver list

### Expected behavior
I should be able to access the Rate / Vote grantor list feature without experiencing a crash

### Actual behavior
When I try to see the list of Giver Rate / Vote in one of the posts, suddenly the application crashes and exits by itself.

### How to reproduce
1. Install and open the application
2. Login
3. Next, go to the Feed page
4. And click on the Giver Rate / Vote list in one of the posts.
![t3t3t](https://user-images.githubusercontent.com/37846899/53695143-6d7c1200-3dea-11e9-85bd-6f5539e51a74.jpg)

*Then you will find problems like mine.*


### Recording Of The Bug

https://youtu.be/634G_WfetDo


### Logcat
```
03-03 19:08:25.900 21983 21983 E AndroidRuntime: FATAL EXCEPTION: main
03-03 19:08:25.900 21983 21983 E AndroidRuntime: Process: com.hapramp, PID: 21983
03-03 19:08:25.900 21983 21983 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.hapramp/com.hapramp.ui.activity.VotersListActivity}: java.lang.NumberFormatException: Invalid double: "0,002"
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2449)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2509)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at android.app.ActivityThread.access$1000(ActivityThread.java:153)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1373)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:5527)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:739)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:629)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: Caused by: java.lang.NumberFormatException: Invalid double: "0,002"
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at java.lang.StringToReal.invalidReal(StringToReal.java:63)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at java.lang.StringToReal.initialParse(StringToReal.java:164)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at java.lang.StringToReal.parseDouble(StringToReal.java:282)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at java.lang.Double.parseDouble(Double.java:301)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at com.hapramp.ui.activity.VotersListActivity.a(Unknown Source)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at com.hapramp.ui.activity.VotersListActivity.m(Unknown Source)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at com.hapramp.ui.activity.VotersListActivity.onCreate(Unknown Source)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:6303)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2402)
03-03 19:08:25.900 21983 21983 E AndroidRuntime: 	... 9 more
```

----
### Environment
Tested on 2 devices:

<table>
<thead>
<tr><th>Device</th><th>Android Version</th>
</thead>
<tbody>
<tr><td>Xiaomi Redmi 4X</td><td>Marshmallow, 6.0.1 mmb29m (CPU: Octa-core Max 1.9 GHz, RAM: 2 GB)</td>
<tr><td>Xiaomi Redmi Note 4</td><td>Marshmallow, 6.0 mRA58K (DECA-CORE MAX 2,11GHz, RAM: 3 gb)</td>

--- a/app/src/main/java/com/hapramp/ui/activity/VotersListActivity.java
+++ b/app/src/main/java/com/hapramp/ui/activity/VotersListActivity.java
@@ -6,7 +6,6 @@ import android.support.v4.content.ContextCompat;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
-import android.view.View;
 import android.widget.ImageView;
 import android.widget.RelativeLayout;
 import android.widget.TextView;
@@ -38,12 +37,9 @@ public class VotersListActivity extends AppCompatActivity {
   RecyclerView userListRecyclerView;
   private ArrayList<Voter> voters;
   private double totalEarning = 0;
-  private Comparator<? super VoterData> votersComparator = new Comparator<VoterData>() {
-    @Override
-    public int compare(VoterData voter1, VoterData voter2) {
-      int comp = voter1.getRshare() > voter2.getRshare() ? -1 : 1;
-      return comp;
-    }
+  private Comparator<? super VoterData> votersComparator = (Comparator<VoterData>) (voter1, voter2) -> {
+    int comp = voter1.getRshare() > voter2.getRshare() ? -1 : 1;
+    return comp;
   };
 
   @Override
@@ -56,12 +52,7 @@ public class VotersListActivity extends AppCompatActivity {
   }
 
   private void attachListeners() {
-    backBtn.setOnClickListener(new View.OnClickListener() {
-      @Override
-      public void onClick(View view) {
-        finish();
-      }
-    });
+    backBtn.setOnClickListener(view -> finish());
   }
 
   private void initList() {
@@ -78,23 +69,26 @@ public class VotersListActivity extends AppCompatActivity {
 
   private ArrayList<VoterData> processVoterData(ArrayList<Voter> voters) {
     ArrayList<VoterData> processedData = new ArrayList<>();
-    long totalRshares = 0;
-    for (int i = 0; i < voters.size(); i++) {
-      totalRshares += voters.get(i).getRshare();
+    try {
+      long totalRshares = 0;
+      for (int i = 0; i < voters.size(); i++) {
+        totalRshares += voters.get(i).getRshare();
+      }
+      for (int i = 0; i < voters.size(); i++) {
+        double percent = voters.get(i).getPercent();
+        long rshare = voters.get(i).getRshare();
+        double voteValue = calculateVoteValueFrom(rshare, totalRshares, totalEarning);
+        voteValue = Double.parseDouble(String.format(Locale.US, "%.3f", voteValue));
+        String voteValueString = voteValue > 0 ? String.format(Locale.US, "$%.3f", voteValue) : "";
+        processedData.add(new VoterData(voters.get(i).getVoter(),
+          String.format(Locale.US, String.format(Locale.US, "%.2f", percent / 100)) + "%",
+          rshare, voteValueString));
+      }
+      Collections.sort(processedData, votersComparator);
     }
-
-    for (int i = 0; i < voters.size(); i++) {
-      double percent = voters.get(i).getPercent();
-      long rshare = voters.get(i).getRshare();
-      double voteValue = calculateVoteValueFrom(rshare, totalRshares, totalEarning);
-      voteValue = Double.parseDouble(String.format("%.3f", voteValue));
-      String voteValueString = voteValue > 0 ? String.format(Locale.US, "$%.3f", voteValue) : "";
-
-      processedData.add(new VoterData(voters.get(i).getVoter(),
-        String.format(Locale.US, String.format(Locale.US, "%.2f", percent / 100)) + "%",
-        rshare, voteValueString));
+    catch (Exception e) {
+      e.printStackTrace();
     }
-    Collections.sort(processedData, votersComparator);
     return processedData;
   }
 

